### ISSUE TITLE
about training code

### ISSUE THREAD

**Issue by ssy-sheshe** [siyuanliii/masa#50]
Happy New Year。When I read the training code，I found that the training data used （sa1b_coco_fmt_500k_bbox_anno.json）did not contain the segmentaion information；however，MASA training pipeline include the part of the segmentation in the paper（training don't need the segmentation?）。I am confused and Can you please explain it to me?

**Comment by siyuanliii** [comment]
Thanks for the question and happy new year! After we generate mask proposals, we can directly extract bounding boxes from those masks. Then, we directly use those bounding boxes to extract object features. The main purpose of segmenting everything in the image is to generate those proposals.

**Comment by ssy-sheshe** [comment]
Thanks for your reply, looking forward to your new achievements.
